@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/engine/fn_engineHandleControl.sqf

```diff
@@ -72,36 +73,82 @@ switch(_control) do {
 			["fza_ah64_switch_flip2", 0.1] spawn fza_fnc_playAudio;
 		};
 	};
+
+	//--------------------ENGINE 1--------------------//
+	//Start Switch
 	case "e1start": {
-		[_heli, 0, ENGINE_CONTROL_STARTER] spawn fza_fnc_engineSetPosition;
-		["fza_ah64_switch_flip4", 0.1] spawn fza_fnc_playAudio;
-	};
-	case "e2start": {
-		[_heli, 1, ENGINE_CONTROL_STARTER] spawn fza_fnc_engineSetPosition;
+		private _eng1State = _heli getVariable "fza_sfmplus_engState" select 0;
+
+		if (_eng1State == "OFF") then {
+			[_heli, 0, "START"] spawn fza_sfmplus_fnc_interactStartSwitch;
```

Comment:
Remove this from the if statement. This is meaning you can flick the start switch when the starter is off, but not when it's already on to turn it back off